### ISSUE TITLE
WP Push Notification using FCM

### ISSUE THREAD

**Issue by myke02** [l3lackcurtains/Flutter-for-Wordpress-App#24]
Hi @l3lackcurtains , I think that Flutter for WP it's a project with great potential!
I had done some tests with Firebase Cloud Message using the firebase console and it's good. But I'm wondering about the possibility to receive Push notification when a post is created in WP or when someone added a comment
There are some plugin for WP Push Notification FCB but I did not found some guides that solve this specific feature with this specific app.

Until now I found this article: https://wordpress.org/support/topic/using-firebase-to-push-notification-from-wordpress/

**Comment by l3lackcurtains** [comment]
I also tried this earlier and didn't find any solutions. There is no proper plugin to integrate firebase with Wordpress. Rather than building a new plugin, I used one signal notification.

So, I have another version of this app that uses one signal notification. By using it, the mobile user gets the notification whenever there is an update in the Wordpress post. 

I have uploaded this version of the app in the codecanyon. It will take a few days to show in codecanyon as I submitted it just yesterday.
